packages/orbit-components/src/Tile/__tests__/index.test.js

```diff
@@ -34,10 +36,15 @@ describe("Tile", () => {
     expect(screen.getByTitle(htmlTitle)).toBeInTheDocument();
     expect(screen.getByRole("button")).toHaveAttribute("tabindex", "0");
     expect(screen.getByText("kek")).toHaveStyle({ padding: "16px" });
+    expect(screen.getByText("kek")).toHaveStyle({
+      borderTop: `1px solid ${theme.orbit.paletteCloudNormal}`,
+    });
 
     userEvent.click(screen.getByRole("button"));
+    fireEvent.keyDown(screen.getByRole("button"), { keyCode: KEY_CODE_MAP.ENTER });
+    fireEvent.keyDown(screen.getByRole("button"), { keyCode: KEY_CODE_MAP.SPACE });
```

Comment:
I think this should also work:
```suggestion
    userEvent.type(screen.getByRole("button"), "{enter}");
    userEvent.type(screen.getByRole("button"), "{space}");
```
[Source](https://github.com/testing-library/user-event/blob/master/README.md#special-characters).